common/lib/xmodule/xmodule/modulestore/tests/test_split_modulestore.py

```diff
@@ -53,7 +54,7 @@ class SplitModuleTest(unittest.TestCase):
     # Snippets of what would be in the django settings envs file
     DOC_STORE_CONFIG = {
         'host': MONGO_HOST,
-        'db': 'test_xmodule',
+        'db': 'test_xmodule_{0}'.format(os.getpid()),
```

Comment:
Further validation: Cale had to do something similar when allowing LMS unit tests to run in parallel: https://github.com/edx/edx-platform/commit/099ce1303aacf178e698be4d06fb1951abcfc3fa .  It looks like the change hasn't been made in CMS; maybe it's needed there too?